osu.Game/Overlays/RestoreDefaultValueButton.cs

```diff
@@ -101,14 +107,16 @@ protected override void OnHoverLost(HoverLostEvent e)
 
         public void UpdateState() => Scheduler.AddOnce(updateState);
 
+        private float currentAlpha => current.IsDefault ? 0f : hovering && !current.Disabled ? 1f : 0.65f;
+        private ColourInfo currentColour => current.Disabled ? Color4.Gray : buttonColour;
```

Comment:
Gray is not showing because color is set in two places: `this` and recently inherited `OsuButton`'s `Background` (set to always yellow).

Reply:
I guess this is a long lost regression irrelevant from this PR, as it still shows as yellow in master:
![CleanShot 2021-08-17 at 06 12 42](https://user-images.githubusercontent.com/22781491/129657988-bdf1db5a-6939-4558-9ed4-60a31aaa72da.png)

This is how it looks with gray, feels *slightly* weird to be honest:
![CleanShot 2021-08-17 at 06 16 49](https://user-images.githubusercontent.com/22781491/129658257-9c21ff0e-2c26-43bc-bfe1-938c702d9659.png)